src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiValueConverter.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiValueConverter : MultiValueConverterExtension, IMultiValueConverter
+	{
+		public MultiBindingCondition ConditionType { get; set; }
+
+		public int Count { get; set; }
+
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values == null)
+				return false;
+
+			var boolValues = values.OfType<bool>();
+
+			if (boolValues.Count() != values.Count())
+				return false;
+
+			var count = boolValues.Where(v => v).Count();
+
+			return ConditionType switch
+			{
+				MultiBindingCondition.Any => count >= 1,
+				MultiBindingCondition.None => count == 0,
```

Comment:
Any: GreaterThan **Count=0**
None: Exact **Count=0**

:)

Do you think we should preserve them for convenient usage?

Reply:
Probably yes, we should. I am just pointing to this case 👍

Reply:
I agree that usage goes above a few fewer lines of code in this case :)

Reply:
I guess Count has 0 as the default value :) So, it should be an equal amount of lines

Reply:
Ok ok, same amount of lines :)